docs/guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_a_filesystem.md

```diff
@@ -2,57 +2,55 @@
 title: How to host and share Data Docs on a filesystem
 ---
 import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
+import TechnicalTag from '/docs/term_tags/_tag.mdx';
 
-This guide will explain how to host and share Data Docs on a filesystem.
+This guide will explain how to host and share <TechnicalTag relative="../../../" tag="data_docs" text="Data Docs" /> on a filesystem.
 
 <Prerequisites>
 
 - [Set up a working deployment of Great Expectations.](../../../tutorials/getting_started/intro.md)
 
 </Prerequisites>
 
-Steps
------
+## Steps
 
-1. **Filesystem-hosted Data Docs are configured by default for Great Expectations deployments created using great_expectations init.**
+### 1. Filesystem-hosted Data Docs are configured by default for Great Expectations deployments created using great_expectations init
```

Comment:
As suggested elsewhere, could these titles be shortened?
```suggestion
### 1. Review defaults and change if desired

Filesystem-hosted Data Docs are configured by default for Great Expectations deployments created using great_expectations init. To create additional... <fill in next sentence>
```

Reply:
I've made and pushed this update.